psalm.xml

```diff
@@ -6,7 +6,7 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="psalm-baseline.xml"
-    findUnusedBaselineEntry="true"
+    findUnusedBaselineEntry="false"
```

Comment:
this gets in the way to much I think for little benefit, especially when trying to Psalm to check one file at a time.